src/lockstep/lockstep_api.py

```diff
@@ -142,21 +144,27 @@ def send_request(self, method: str, path: str, body: object, query_params: objec
         query_params : object
             The list of query parameters for the request
         """
+        start_time = time.time()
+
         if query_params:
             url = urllib.parse.urljoin(self.serverUrl, path) + "?" + urllib.parse.urlencode(query_params)
         else:
             url = urllib.parse.urljoin(self.serverUrl, path)
-        
+
+        execution_time = time.time() - start_time
         headers = {"Accept": "application/json",
                    "SdkName": self.sdkName,
                    "SdkVersion": self.sdkVersion,
                    "MachineName": self.machineName,
-                   "ApplicationName": self.applicationName}
+                   "ApplicationName": self.applicationName,
+                   "RoundTripTime": execution_time,
+                   "ServerDuration": self.serverDuration}
```

Comment:
`serverDuration` is a value that is received in the response, not sent in the header.

We should be parsing the response to retrieve this value.